packages/data-provider/src/config/logger.ts

```diff
@@ -42,22 +42,22 @@ export const getLogger = (config: LoggerConfig): Logger => {
 		transports: transportList,
 	});
 
-	const log = (...message: string[]) => {
+	const log = (...message: unknown[]) => {
 		const fullMessage = message.join(' - ');
 		return fullMessage;
 	};
 
 	return {
-		debug: (...messages: any[]) => {
+		debug: (...messages: unknown[]) => {
 			return logger.debug(log(...messages));
 		},
-		warn: (...messages: any[]) => {
+		warn: (...messages: unknown[]) => {
 			return logger.warn(log(...messages));
 		},
-		info: (...messages: any[]) => {
+		info: (...messages: unknown[]) => {
 			return logger.info(log(...messages));
 		},
-		error: (...messages: any[]) => {
+		error: (...messages: unknown[]) => {
```

Comment:
hmmmm I see this is a place where we were using `any` in a logical way. `unknown` works fine but looks weird.

Reply:
Will leave `unknown` as it's recommended over `any` when it's possible